data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1847 @@ mission "Stone of our Fathers 5"
+			`	Or you can try to repair the Ice-Crawler and get it running again. In which case you wouldn't be able to use its parts to boost your comm signal.`
+			choice
+				`	(Cannibalize the Ice-Crawler and contact your ship.)`
+					goto stickmud
+				`	(Give up trying to contact your ship and fix the Ice-Cutter.)`
+					goto fixme
+			label stickmud
+			`	You have much more faith in your ship than some antique vehicle. You spend hours ripping once-vital machinery out of the Ice-Crawler in order to make a signal booster for your comms. Eventually cobbling together a functional booster and programming a repeating burst S.O.S. message into it.`
+			`	You're certain your ship will eventually pick it up. Unfortunately, the damage you've done to the Ice-Crawler has affected its heater, which gives out during your eighth hour of waiting.`
+			`	In the meantime, you put on the jacket Hjlod left you and stuff it with rags and trash in a bid to keep warm.`
+			choice
+				`	(Shivering, you try to ignore the cold and get some sleep.)`
+			`	It has been 19 hours since the avalanche, and you've lost feeling in your hands, and feet. There are no rations and what stale warmth was once here is now gone.`
+			`	Hopefully you didn't make a mistake waiting here. You know you made a mistake by going on this miserable tour. You just want to be warm again, and you certainly don't want to die here.`
+			`	You begin to fade in and out of consciousness, when suddenly you feel a deep vibration, and hear a low rumble from outside.`
+			`	There is no memory of what happens next. But somehow you must have crawled out of that ice buried Ice-Crawler, through the snow, and into the hatch of your ship. Because the next thing you do remember is laying in your ship, warming up, and being glad that the ordeal is over.`
+				decline
+			label fixme
+			`	After hours of tinkering with the guts of the machine you've managed to repair most of it, and all it cost you was one bad burn on your arm. Now, what it comes down to is a sequence of wires. If you can plug these in correctly, the Ice-Crawler should start back up.`
+			choice
+				`	(White, Green, Blue.)`
+					goto boom
+				`	(Green, White, Blue.)`
+					goto greatsuccess
+				`	(Blue, Green, White.)`
+					goto boom
+			label boom
+			` 	You're confident that this is the correct sequence of wires. You plug each one in and start the Ice-Crawler up.`
+			`	The vehicle hums to life.`
+			choice
+				`	(Pump your fist.)`
+					goto fist
+				`	(Cheer.)`
+					goto cheer
+			label fist
+			` 	In a moment of levity you pump your fist into the air.`
+			`	Unfortunately, your fist does not make it very far. As the Ice-Crawler, with its criss-crossed wires, has a critical energy build up that results in an instantaneous completely deadly explosion.`
+			`	The Ice-Crawler, your entire body, your fist, and more than a couple hundred tons of snow, are all incinerated in a fraction of a second.`
+			scene "scene/smeer"
+				die
```

Comment:
```suggestion
			scene "scene/smeer"
			`	The Ice-Crawler, your entire body, your fist, and more than a couple hundred tons of snow, are all incinerated in a fraction of a second.`
				die
```
Looks like there's a bug where you can't die right after a `scene`, so we gotta re-arrange it to murder \<first> \<last> appropriately.